features/built_in_matchers/respond_to.feature

```diff
@@ -133,7 +133,7 @@ Feature: `respond_to` matcher
   Given a file named "respond_to_matcher_unlimited_argument_checking_spec.rb" with:
       """ruby
       class MyClass
-        def greet(message = 'Hello', *people)
+        def greet(message='Hello', *people)
```

Comment:
I meant the blank lines, the white space in the arguments is necessary per our style guidelines! Sorry!